plugins/de.ovgu.featureide.fm.core/library/subant_build_jar.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Are these changes intended? How are they related to the issue?

Reply:
This seems to have happened due to cafab33 from Sebastian. I don't know why it's in here, it is related to #1100.